requirements-dev.in

```diff
@@ -1,3 +1,5 @@
+-c ${CONSTRAINTS}
+-r requirements.in
```

Comment:
how about addind `botocore-types` and `boto3-types` to dev? (or if they're called `types-*`, I can never remember...)

Reply:
they are all pulled in from boto3-stubs[essential, stepfunctions]

Reply:
Cool. That sounds more convenient.